pipelines/jobs/configurations/jdk11u.groovy

```diff
@@ -21,7 +21,8 @@ weekly_release_scmReferences=[
         "hotspot"        : "",
         "openj9"         : "",
         "corretto"       : "",
-        "dragonwell"     : ""
+        "dragonwell"     : "",
+        "bisheng"	 : ""
```

Comment:
Do we want this lined up?

Reply:
Yeah - the online merge conflict resolution wasn't doing a good job of showing me the spacing properly as I recall ...

Reply:
Done - it was tab characters that messed with it - replaced with spaces now